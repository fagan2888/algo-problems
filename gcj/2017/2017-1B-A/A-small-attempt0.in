99
2525 1
2400 5
300 2
120 60
60 90
444461490 1
307501350 7148
809156948 1
413641954 7558
211231445 1
27922335 7021
500984270 1
222465901 6818
728317566 1
357843460 7646
370603308 1
146373963 8731
1000000000 1
999999999 1
501169380 1
171407215 7411
115304536 1
18994917 2867
632118401 1
468797480 160
260753973 1
51405370 7733
445496400 1
146718134 2345
592326380 1
156213112 409
37871670 1
24208334 6012
1000000000 2
2 2
1 3
697404202 1
315965799 6585
1000000000 2
999999998 2
999999997 3
440974324 1
317988347 5497
431873002 1
117400261 355
867253024 1
835897135 2295
314021464 1
29959767 5683
776794900 1
449206526 4953
39671785 1
1088903 6471
87115763 1
48963666 7384
306887689 1
258289995 9151
471216583 1
172965963 4415
359836742 1
333758054 8113
148072109 1
1785160 5989
798750343 1
740211353 3597
995208768 1
933844929 4045
234235738 1
172664100 2627
355989187 1
59603696 635
838916529 1
642315134 5312
123150339 1
111119768 2302
427034002 1
30305466 8841
1000000000 2
12345676 2
12345675 3
60298908 1
55632750 2951
90561446 1
67557162 9949
879233775 1
595378180 1247
467054865 1
352078160 2998
232228254 1
31906928 9179
745530415 1
453628354 9643
1000000000 1
1 10000
587916320 1
314476363 1650
2 1
1 1
342302510 1
161130700 4931
54792508 1
51763141 4193
1000000000 2
999999996 3
999999997 2
92455639 1
56341266 7362
666863211 1
122640299 9057
182774182 1
18814328 160
434867380 1
387252859 7455
1000000000 2
100 10000
200 10000
329108372 1
190763817 9452
78937694 1
17666195 8098
171637308 1
31928677 9281
53170857 1
11240965 1316
327556798 1
295761857 4606
633060260 1
273182995 729
103884003 1
60603957 2356
983877852 1
137724849 7335
370975381 1
276201238 7039
471968609 1
387687018 2497
229558787 1
35060886 3679
810147067 1
729579357 4320
1000000000 1
999999999 10000
304074909 1
258358937 1241
718417182 1
165977267 9245
328491529 1
196689895 7336
587348605 2
538167563 8690
570189796 3630
264328253 1
228861465 217
1000000000 1
1 1
990512963 1
149176012 1967
200554101 1
81165874 3453
285946926 1
71986640 4276
2 1
1 10000
291455884 1
116661996 7546
586003083 1
262841723 3733
1000000000 2
998000000 10000
999000000 10000
684156885 1
645509594 1832
6562989 1
2601697 9073
629728096 1
270376145 3522
996979986 1
453157948 4779
1000000000 2
999999999 2
999999998 3
879300288 1
378706562 3722
200000 2
5000 6
5001 1
1000000000 2
999000000 1
998000000 1
474420790 1
24884287 2250
506175851 1
332765195 5342
13689108 1
3163728 3134
643551696 2
239209539 5233
420319322 6991
63430510 1
21986934 9769
772386861 1
32642170 1716
273646716 1
253185183 4185
137164130 1
60978278 2367
576801220 1
247321211 3040
973701903 1
209833794 7840
