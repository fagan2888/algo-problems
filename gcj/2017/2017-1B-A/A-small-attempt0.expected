Case #1: 101.000000
Case #2: 100.000000
Case #3: 23196.608375
Case #4: 15462.392844
Case #5: 8090.465200
Case #6: 12263.861680
Case #7: 15031.323430
Case #8: 14430.481800
Case #9: 1000000000.000000
Case #10: 11263.166820
Case #11: 3432.451588
Case #12: 619.265086
Case #13: 9631.831521
Case #14: 3496.536318
Case #15: 555.501305
Case #16: 16663.901118
Case #17: 2.000000
Case #18: 12039.707156
Case #19: 1000000000.000000
Case #20: 19709.855694
Case #21: 487.530065
Case #22: 63475.977035
Case #23: 6282.381605
Case #24: 11744.815888
Case #25: 6653.627397
Case #26: 16860.483291
Case #27: 57787.294229
Case #28: 6975.412872
Case #29: 111944.108839
Case #30: 6062.084601
Case #31: 49080.193966
Case #32: 65602.470969
Case #33: 9993.843005
Case #34: 762.699729
Case #35: 22666.800518
Case #36: 23564.307993
Case #37: 9516.350020
Case #38: 2.025000
Case #39: 38134.601852
Case #40: 39166.436402
Case #41: 3862.543268
Case #42: 12178.384180
Case #43: 10641.019536
Case #44: 24628.636630
Case #45: 10000.000010
Case #46: 3547.623173
Case #47: 2.000000
Case #48: 9316.535927
Case #49: 75839.271387
Case #50: 666666666.666667
Case #51: 18847.299781
Case #52: 11097.989388
Case #53: 178.359936
Case #54: 68087.134971
Case #55: 10000.001000
Case #56: 22485.397652
Case #57: 10432.867752
Case #58: 11402.057583
Case #59: 1668.805820
Case #60: 47451.782080
Case #61: 1282.384230
Case #62: 5655.047388
Case #63: 8528.887824
Case #64: 27552.828485
Case #65: 13982.954079
Case #66: 4342.189674
Case #67: 43439.677377
Case #68: 10000000000000.000000
Case #69: 8254.379062
Case #70: 12022.604934
Case #71: 18283.641740
Case #72: 103781.033705
Case #73: 1617.266015
Case #74: 1.000000
Case #75: 2315.765397
Case #76: 5800.515915
Case #77: 5714.654240
Case #78: 20000.000000
Case #79: 12582.397050
Case #80: 6769.217424
Case #81: 5000000.000000
Case #82: 32431.132452
Case #83: 15031.964116
Case #84: 6171.950223
Case #85: 8761.261994
Case #86: 1500000000.000000
Case #87: 6537.748082
Case #88: 1.025646
Case #89: 500.000000
Case #90: 2374.549720
Case #91: 15592.994447
Case #92: 4076.020483
Case #93: 8328.852104
Case #94: 14951.717781
Case #95: 1791.720670
Case #96: 55968.998338
Case #97: 4261.519523
Case #98: 5321.948710
Case #99: 9993.640040
